frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -525,9 +525,7 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 plugins.push(CorsPlugin)
             }
 
-            if (values.featureFlags[FEATURE_FLAGS.SESSION_REPLAY_CANVAS]) {
```

Comment:
we should tidy up `FEATURE_FLAGS.SESSION_REPLAY_CANVAS` from the constants list too :)

Reply:
Good point, I'll follow up with a PR to remove any other flags we've released too